docs/vc-http-api.yml

```diff
@@ -428,5 +435,10 @@ components:
           $ref: "#/components/schemas/VerifiablePresentation"
         options:
           $ref: "#/components/schemas/LinkedDataProofOptions"
+    ProoflessVerifyPresentationRequest:
+      type: object
+      properties:
+        presentation:
+          $ref: "#/components/schemas/Presentation"
```

Comment:
Let's get rid of the `Proofless` and plain 'ol `Presentation` language -- these are all "VerifiablePresentations" --- https://w3c.github.io/vc-data-model/#presentations-0  -- we were clear in the spec that you don't need to have a proof on a VP for it to be a valid VP... you just can't verify the presentation (but you can still verify the VCs in the presentation.

Reply:
so the actual object passed is a `Presentation` object.  Proofless vs Presentation is just how that unique element is identified in the schema description to identify the unique combination of parameters.  I will work up an option here and propose something.  

@msporny My goal here was to implement in such a way that the existing YAML still held with no changes - am I fine to propose a change to the YAML, or could that delay getting this capability in?  

if we can change existing elements, this section can be restructured a bit for clarity, and at that point I would also issue a separate PR to clean up error handling conventions and docs embedded in the spec as well.  Big thing is, that I want to make sure we get all changes in in time for the impending code freeze, in a manner that does not change any existing implementations for vendors.  If we can make minor changes to parameter naming in the docs this whole pr could have been cleaner from the start.

Reply:
Happy to fix this later... don't want to block people that need it.

My point was that we're starting to use language that we either specifically decided not to use during the VCWG, or we're inventing new language that might cause people to go to the VC spec to look up the concept only to find nothing there wrt. class being described. We can bike shed the names later as long as all of us understand that it needs to be done and the current names we're using are just placeholders.

Reply:
agreed entirely to this.  there are a few naming things that would be very helpful to revisit, as well as structural items int he openapi spec that we could address that will make sure things are clean, enterprise friendly, and self documenting.